pkg/trait/util.go

```diff
@@ -388,34 +388,43 @@ func KameletBindingsHaveSameTraits(i1 *v1alpha1.KameletBinding, i2 *v1alpha1.Kam
 }
 
 // IntegrationAndBindingSameTraits return if traits are the same.
+// The comparison happen for a subset of traits as during the trait processing, some traits may be
```

Comment:
nit: happen -> happens or "is done for the subset of traits defines on the binding"